ra-optimisation/search-tree-rao/src/main/java/com/powsybl/openrao/searchtreerao/castor/algorithm/AutomatonSimulator.java

```diff
@@ -249,7 +249,7 @@ public Set<NetworkAction> getActivatedNetworkActions() {
     TopoAutomatonSimulationResult simulateTopologicalAutomatons(State automatonState, Network network, PrePerimeterSensitivityAnalysis preAutoPstOptimizationSensitivityAnalysis) {
         // -- Apply network actions
         // -- First get forced network actions
-        Set<FlowCnec> flowCnecs = crac.getFlowCnecs();
+        Set<FlowCnec> flowCnecs = crac.getFlowCnecs(automatonState);
```

Comment:
should we also check the curative cnecs on the same contingency?

Reply:
(to check with Sebastien)

Reply:
It is a good question... I would say no, but still have to check